CMakeLists.txt

```diff
@@ -16,21 +16,16 @@ cmake_minimum_required(VERSION 3.17.2)
 
 project(VUL LANGUAGES CXX)
 
-include(GNUInstallDirs)
+string(COMPARE EQUAL ${CMAKE_CURRENT_SOURCE_DIR} ${CMAKE_SOURCE_DIR} VUL_IS_TOP_LEVEL) # Remove when min is 3.21
 
-set_property(GLOBAL PROPERTY USE_FOLDERS ON) # Remove when min is 3.26, see CMP0143 
+set_property(GLOBAL PROPERTY USE_FOLDERS ON) # Remove when min is 3.26, see CMP0143
 
-option(VUL_TESTS "Build utility libraries tests")
-
-set(CMAKE_CXX_STANDARD 17)
-set(CMAKE_CXX_STANDARD_REQUIRED ON)
-set(CMAKE_CXX_EXTENSIONS OFF)
 set(CMAKE_POSITION_INDEPENDENT_CODE ON)
-set(CMAKE_CXX_VISIBILITY_PRESET "hidden")
-set(CMAKE_VISIBILITY_INLINES_HIDDEN "YES")
```

Comment:
This code only does something for dynamic libraries (.so / .dll / .dylib). If we are only shipping a static library we don't need this.